tz.h

```diff
@@ -51,7 +51,13 @@
 #    error "Timezone mapping is not required or not implemented for this platform."
 #  endif
 #  if !defined(TIMEZONE_FILES) && !defined(TIMEZONE_RULES)
-#    define TIMEZONE_FILES 1
+#    if !defined(__APPLE__)
+#      define TIMEZONE_RULES 1
+#      define TIMEZONE_FILES 0
+#    else
+#      define TIMEZONE_RULES 0
+#      define TIMEZONE_FILES 1
+#    endif
```

Comment:
Quite right! That's what I get for working on it right before bed. I ammended my commit.